include/fw_info.rst

```diff
@@ -29,6 +29,9 @@ It includes the following information:
 * The single, monotonically increasing version number of the image.
 * The address through which to boot into the firmware (the vector table address).
   This address is not necessarily the start of the image.
+* A value that must be 0xFFFFFFFF.
```

Comment:
Can you elaborate? If this check is spoofed during validation, the signature check will still fail.

Reply:
Since 0xFFFFFFFF is the  value of erased memory, my gut feeling is that it would be easier to spoof this value somehow, compared to a random value 0x2B68C12A

Reply:
Ok, I think I understand. For it to work in the OTP region, it would have to be something like 

    #define VALID_VAL 0x2B68FFFF
    #define INVALID_VAL 0x2B680000

Reply:
Implemented now. I also put the valid value in Kconfig.

Reply:
Does this need a doc update?

Reply:
Yes, thanks. Updated docs now.